mlflow/server/js/src/experiment-tracking/routes.js

```diff
@@ -7,6 +7,16 @@ class Routes {
     return `/experiments/${experimentId}`;
   }
 
+  static getExperimentByUser(experimentId, user_id) {
+    const searchString = `searchFilter=user_id:${user_id}`
+    return `/experiments/${experimentId}/${searchString}`;
+  }
+
+  static getExperimentByLifeCycle(experimentId, lifeCycleStage) {
+    const searchString = `searchFilter=lifeCycleStage:${lifeCycleStage}`
```

Comment:
```suggestion
    const searchString = `lifecycleFilter=${lifeCycleStage}`
```

Reply:
Hi @harupy , with this change in formatting in line 16, would you like me to format line 11 similarly, like,
const searchString = `user_id=${user_id}`;?

Reply:
```suggestion
```